src/view/Pane.js

```diff
@@ -161,15 +161,44 @@ define(function (require, exports, module) {
         InMemoryFile        = require("document/InMemoryFile"),
         ViewStateManager    = require("view/ViewStateManager"),
         MainViewManager     = require("view/MainViewManager"),
+        PreferencesManager  = require("preferences/PreferencesManager"),
         DocumentManager     = require("document/DocumentManager"),
         CommandManager      = require("command/CommandManager"),
         Commands            = require("command/Commands"),
         Strings             = require("strings"),
+        StringUtils         = require("utils/StringUtils"),
         ViewUtils           = require("utils/ViewUtils"),
         ProjectManager      = require("project/ProjectManager"),
         paneTemplate        = require("text!htmlContent/pane.html");
     
     /**
+     * Internal pane id
+     * @const
+     * @private
+     */
+    var FIRST_PANE          = "first-pane";
+
+    /**
+     * Internal pane id
+     * @const
+     * @private
+     */
+    var SECOND_PANE         = "second-pane";
+
+    // Define showPaneHeaderButtons, which controls when to show close and flip-view buttons
+    // on the header. Possible values "hover", "always" and "never"
+    PreferencesManager.definePreference("pane.showPaneHeaderButtons", "string", "hover", {
+        description: Strings.DESCRIPTION_SHOW_PANE_HEADER_BUTTONS
```

Comment:
You seems to have forgotten to add `"hover" "always"  "never"` to values array here so it can be shown in the hints.